src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -41,6 +41,9 @@ public Drivetrain()
   // METHODS // 
   
   public void moveMotors (double speed, double turn) {
+    speed = Math.pow(speed, 2);
+    turn = Math.pow(turn, 2);
```

Comment:
This does indeed reduce the sensitivity.  However, it also makes the values always positive, which will break this code which relies on positive and negative vectors:
```
rightMotor.set(speed + turn);
leftMotor.set(speed - turn);
```